### DISCUSSION TITLE
Keyboard Shortcuts

### DISCUSSION THREAD

**Discussion by meteos77** [textbrowser/biblioteq#132] [Ideas]
suggestion : one more page with a list of keyboard shortcuts
fonts-linuxlibertine - Famille de polices Linux Libertine

[List_Keyboard_Shortcuts-GB.pdf](https://github.com/textbrowser/biblioteq/files/7653801/List_Keyboard_Shortcuts-GB.pdf)
[List_Keyboard_Shortcuts-GB.odt](https://github.com/textbrowser/biblioteq/files/7653802/List_Keyboard_Shortcuts-GB.odt)

**Comment by textbrowser** [comment]
A new release is much needed. After the SQLite merge, a new release. Then a focus on your import.

**Reply by meteos77** [reply]
super I test this and give my feedback

**Reply by meteos77** [reply]
I didn't manage to make the merge?
the baby is out :-) ?

**Reply by textbrowser** [reply]
The SELECTs and the INSERTs remain; as well as their infrastructure.

**Reply by meteos77** [reply]
I was talking about the merge interface

![merge-ui](https://user-images.githubusercontent.com/56431039/144719006-2e2d9f3a-b49d-45f4-9e97-01053f2d8ccd.png)

**Comment by textbrowser** [comment]
What about it?

**Reply by meteos77** [reply]
it doesn't make the merger?

**Reply by textbrowser** [reply]
I answered that above.

"The SELECTs and the INSERTs remain; as well as their infrastructure."

**Reply by meteos77** [reply]
your answer is not understandable for me?

**Reply by textbrowser** [reply]
The merge requires many INSERT and SELECT statements because I cannot find an easy way of performing a SELECT from one database and an INSERT into another database using a single statement.

**Reply by textbrowser** [reply]
There are also unique integer fields which cannot be automatically adjusted by SQLite.

**Reply by meteos77** [reply]
thanks for the explanation
if this functionality is not possible with SQLite, you can't do miracles yet :-) we will think about how to do it differently

**Reply by meteos77** [reply]
last time you didn't understand the logic of deleting 1000 documents at once, the process was not fair.

ideas: 

1 - Selection of documents with SQL SELECT 
2 - Export to CSV ;
3 - SQL DELETE of the same documents
4 - Creation of a new database
5 - Import the CSV into the new database.

In the end, do we get the same thing as the merge SQL version?


other options : 
Can we create in the same "file" database two "pseudo databases" ?
An active database : Library
a non-active pseudo base : Archive-Library1 Archive-Library2 ...
and have a selection on the pseudo base we want to view/work on

advantage : it's the same database : if it solves the merge problem ???
disadvantage : the database will never be thinned this way but we will work with portions as if they were separate file bases

Translated with www.DeepL.com/Translator (free version)

**Reply by textbrowser** [reply]
The merge is not related to this. The merge literally copies the contents of N databases to a destination database.

**Comment by textbrowser** [comment]
![Screenshot_20211204_173101](https://user-images.githubusercontent.com/10701156/144726764-bb12bdc1-8ecb-4d1b-bcb8-505ee67100fc.png)

:)
